src/main/kotlin/gg/skytils/skytilsmod/features/impl/dungeons/catlas/core/CatlasElement.kt

```diff
@@ -180,11 +173,13 @@ object CatlasElement : GuiElement(name = "Dungeon Map", x = 0, y = 0) {
             val yOffsetName = (namePos.second / 2f) * size
 
             val color = if (CatlasConfig.mapColorText) when (room.state) {
-                RoomState.GREEN -> 0x55ff55
-                RoomState.CLEARED -> 0xffffff
-                RoomState.FAILED -> 0xff0000
-                else -> 0xaaaaaa
-            } else 0xffffff
+                RoomState.GREEN -> Color(85,255,85).rgb
```

Comment:
I believe this uses more memory (unless there is a JVM optimization) especially since this is run every frame.